cirq/work/pauli_sum_collector_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cirq
+
+
```

Comment:
you've got an ```assert pauli_string``` but no test that triggers it.

Reply:
That assert is actually never supposed to fail; it's documentation. Added a test that uses an identity term, and ensured it ends up in the output.